docs/book/src/user/quick-start.md

```diff
@@ -434,82 +474,84 @@ spec:
 
 <h1>Action Required</h1>
 
-These examples include environment variables that you should substitute before creating the resources.
+This quick start assumes the following vSphere environment which you should replace based on your own environment:
+
+```
```

Comment:
Hi @andrewsykim,

Please see my earlier comment about the formatting of these properties.